### PR TITLE
Load all of a league's teams and their ranking by a single sql command

### PR DESCRIPTION
**Pull request by namxam**
Instead of calculating scores in ruby, we can harness the power of SQL and do all of it in our database.

### CODE REVIEW

**Comment by namxam** [comment]
@DonSchado I guess you will have some objections :smile:

File: app/models/league.rb [hunk 1]
```diff
@@ -63,6 +65,16 @@ def active_user_ranking
       .reorder('quota DESC')
   end
 
+  def team_ranking
+    teams
+    .select("teams.*, COUNT(matches.*) AS games_played, #{BASE_SCORE} + SUM(CASE WHEN matches.loser_team_id = teams.id THEN matches.difference * -1 ELSE matches.difference END) AS score")
```
Comment (DonSchado) [inline app/models/league.rb]: why `* -1` ? 
don't understand the query I think...

**Comment by DonSchado** [comment]
@namxam Love it! :shipit: 

(where are the specs btw.?)

File: app/models/league.rb (hunk 1 above)
Comment (namxam) [inline app/models/league.rb]: ```CASE WHEN matches.loser_team_id = teams.id THEN matches.difference * -1```… meaning that if the team lost the match, it should lower the teams total score by the match score.

**Pull request merged**